05-dijkstras/03-dijkstra.md

```diff
@@ -2,6 +2,22 @@
 
 <iframe src="https://adaacademy.hosted.panopto.com/Panopto/Pages/Embed.aspx?pid=06222ed2-1765-4cd5-9424-afe80148c8c9&autoplay=false&offerviewer=true&showtitle=true&showbrand=true&captions=true&interactivity=all" height="405" width="720" style="border: 1px solid #464646;" allowfullscreen allow="autoplay"></iframe>
 
+### !callout-star
+
+## Practical Applications!
+
+Some common practical applications of Djikstra's Algorithm can be found below. As you learn more, try to imagine how it could be used to implement the following:
+
+- Digital Mapping Services
+- Social Networking applications
```

Comment:
Nitpick - capitalize "Applications" to match capitalization of other bullet points.

Reply:
And Done!